packages/libs/deer/src/error/mod.rs

```diff
@@ -83,6 +104,78 @@ impl Id {
     }
 }
 
+// TODO: most likely (in 0.2) we want to actually have a proper schema
+pub struct Schema {
+    ty: String,
+    other: BTreeMap<String, Box<dyn erased_serde::Serialize + Send + Sync>>,
```

Comment:
Is there a specific reason to use `BTreeMap` over `HashMap`?

Reply:
Yes, `HashMap` is only available on `no-std` over the hashbrown crate, while `BTreeMap` is available on `alloc`.